src/scripting/lua_map.cc

```diff
@@ -2366,11 +2406,20 @@ int LuaImmovableDescription::get_becomes(lua_State* L) {
 /* RST
    .. attribute:: terrain_affinity
 
-         returns the terrain affinity values for this immovable
+      (RO) For trees: A :class:`table` containing numbers labeled as pickiness (uint),
```

Comment:
Bushes also have terrain affinity

Reply:
"uint" should probably have type markup?

Reply:
Since Lua has no `uint` i removed them all.